src/qt/tapyrus.cpp

```diff
@@ -543,7 +543,7 @@ static void SetupUIArgs()
     gArgs.AddArg("-resetguisettings", "Reset all settings changed in the GUI", false, OptionsCategory::GUI);
     gArgs.AddArg("-rootcertificates=<file>", "Set SSL root certificates for payment request (default: -system-)", false, OptionsCategory::GUI);
     gArgs.AddArg("-splash", strprintf("Show splash screen on startup (default: %u)", DEFAULT_SPLASHSCREEN), false, OptionsCategory::GUI);
-    gArgs.AddArg("-uiplatform", strprintf("Select platform to customize UI for (one of windows, macosx, other; default: %s)", BitcoinGUI::DEFAULT_UIPLATFORM), true, OptionsCategory::GUI);
+    gArgs.AddArg("-uiplatform", strprintf("Select platform to customize UI for (one of windows, macosx, other; default: %s)", TapyrusGUI::DEFAULT_UIPLATFORM), true, OptionsCategory::GUI);
 }
 
 #ifndef BITCOIN_QT_TEST
```

Comment:
This is also the target?

Reply:
Yes, this is intentional.  `DEFAULT_UIPLATFORM` is a static string defined inside the class `TapyrusGUI` . We changed the class name.

`static const std::string DEFAULT_UIPLATFORM;`

Reply:
oh, I'm talking about

```cpp
#ifndef BITCOIN_QT_TEST
```

Reply:
sorry! you mean change BITCOIN_QT_TEST to TAPYRUS_QT_TEST? Most of the include guards and conditional compilation are still starting with BITCOIN_ Maybe I'll make a separate PR for these and the their makefile changes?

Reply:
ok.  Let's make a separate PR for this.